SU2_CFD/include/fluid/CSU2TCLib.hpp

```diff
@@ -72,6 +72,25 @@ class CSU2TCLib : public CNEMOGas {
   C3DDoubleMatrix Omega00,       /*!< \brief Collision integrals (Omega(0,0)) */
   Omega11;                       /*!< \brief Collision integrals (Omega(1,1)) */
 
+  /*--- Implicit variables ---*/
```

Comment:
This may cause issues.  These values will be set in ComputeNetProductionRates/ComputeEveSourceTerm and "saved" for the Jacobian computations at an attempt for efficiency.